rxnetty-common/src/main/java/io/reactivex/netty/channel/ChannelOperations.java

```diff
@@ -83,7 +83,7 @@ public Boolean call(FileRegion next) {
      *
      * @param msgs Message stream to write.
      * @param flushSelector A {@link Func1} which is invoked for every item emitted from {@code msgs}. Channel is
-     * flushed, iff this function returns, {@code true}.
+     * flushed, if this function returns, {@code true}.
```

Comment:
The `iff` here is intentional. [Ref](http://www.merriam-webster.com/dictionary/iff)